pages/linux/wall.md

```diff
@@ -1,7 +1,6 @@
 # wall
 
 > Write a message on the terminals of users currently logged in.
-> Only available to super-user.
```

Comment:
I think a "the" is needed here such that it reads "Only available to the super-user".